android/jni/com/mapswithme/maps/ugc/UGC.cpp

```diff
@@ -88,7 +88,7 @@ class JavaBridge
     jobjectArray jratings = static_cast<jobjectArray>(env->GetObjectField(ugcUpdate, m_ratingArrayFieldId));
     int const length = env->GetArrayLength(jratings);
     std::vector<ugc::RatingRecord> records;
-    records.reserve(length);
+    records.reserve((unsigned int) length);
```

Comment:
We do not use C-style casts.
Please use static_cast.

Reply:
Fixed